test/e2e/framework/internal/unittests/cleanup/cleanup_test.go

```diff
@@ -122,7 +122,17 @@ func TestCleanup(t *testing.T) {
 	logger, _ := ktesting.NewTestContext(t)
 	klog.SetLogger(logger)
 
-	apiServer := testapiserver.StartAPITestServer(t)
+	// disable some options to save some during bootstrap
```

Comment:
how much time do these save? unless it's significant, I'd prefer not to piece-mill disable default admission plugins and featurers

Reply:
added in the commit some info

> Local tests on my computer shows an improvement from an average of 16 seconds,
with a +-0.5 marging error, to 12 seconds with the same error marging.

but you are right, this is not the main problem here, jobs that run on environments that are not able to boot an apiserver+etcd are not reliable and they have to be dimensioned accordingly ...